documentation/src/main/asciidoc/user_guide/cross_site_replication.adoc

```diff
@@ -213,6 +213,47 @@ expired in clusters on other sites.
 {brandname} cannot determine if cache entries have reached the idle timeout in
 clusters on other sites.
 
+[[xsite_concurrent_writes]]
+==== Deadlocks and Conflicting Entries
+In active/active configurations, concurrent writes to the same key results in
+deadlocks or conflicts.
+
+Active/active configurations are those in which both the local and the backup
+sites replicate data to each other. For example, the site "LON" backs up to the
+site "SFO" and "SFO" also backs up to "LON".
+
+The following example illustrates how deadlocks and conflicts occur with
+concurrent writes in active/active configurations:
+
+[source,txt,options="nowrap"]
+----
+include::xsite_conflicts.txt[]
+----
+
+<1> `x` represents a key in the data store.
+<2> client writes `x=3` on "LON". "LON" replicates to "SFO".
+<3> client writes `x=5` on "SFO". "SFO" replicates to "LON"
+<4> client writes `x=6` on "LON" at the same time that a client writes `x=8` on "SFO".
```

Comment:
The sequence is not clear, it does not show when X is locked where.
I would have the sequence like
client write x=3 to LON
LON cache lock x
LON replicate to SFO and lock x(SFO)
......
For <4> you can see that
LON lock x (local)
SFO lock x (local)
LON repl x and try to lock x(SFO)
SFO repl x and try to lock x(LON) --> deadlock

Reply:
@wfink I've updated. to me it makes it somewhat more confusing to try and show where the locks exist.